CONTRIBUTORS.md

```
@@ -0,0 +1,12 @@
+## 💫 Contributors
+
+Thanks to our amazing contributors! 💖
+
+<a href="https://github.com/topoteretes/cognee/graphs/contributors">
+  <img src="https://contrib.rocks/image?repo=topoteretes/cognee" />
```

markdownlint-cli2 (0.17.2)

Images should have alternate text (alt text)
null

(MD045, no-alt-text)